README.md

```diff
@@ -246,6 +246,11 @@ Billed Resources and Infrastructure:
 
 **Total Average Monthly Cost**: $60.00
 
+This is the ongoing cost of storing new meetings as they occur once your instance is deployed.
```

Comment:
$1.24 was the cost for the [single 1:58:59 video](https://smai-f.github.io/missoula-council-data-project/#/events/311d163376be) I pushed to my otherwise empty db.

It was a bit of sticker shock when I realized we're going to have to spend around $150 to get all the meeting vids from 2022 pushed into our instance. It's no biggie for us, but would have been unpleasant to realize this after doing the work to deploy an instance if we couldn't cover the cost. 

I'm an outsider and am positive I don't have a lot of the context around all that went into this section, but feel strongly enough to at least try to get it included 😄

Reply:
Yeah that's fair. I also just updated the pricing on the README a bit. The old pricing was from a year or so ago and I think both our increased data requirements and Google pricing changes maybe increased it? Not sure.

Glad to hear it's not a problem but do let me know if it is.

No closed caption files for missola unfortunately?

---

Also how many meetings / total time per month does missoula have? The Seattle estimate was for about 10 hours of meetings a week give or take.